src/login.rs

```diff
@@ -158,8 +181,12 @@ impl Login {
             AuthorizationState::WaitOtherDeviceConfirmation(_) => {
                 todo!()
             }
-            AuthorizationState::WaitRegistration(_) => {
-                todo!()
+            AuthorizationState::WaitRegistration(data) => {
+                self_.show_tos_popup.set(data.terms_of_service.show_popup);
+                self_.tos_text.replace(data.terms_of_service.text.text);
```

Comment:
The returned TOS here is a formatted text, which means that it can contain text that needs to be parsed to be displayed with the correct format. Now, I've tested to see if there's actually some entries to parse and it doesn't seem to be the case, but I'd still prefer to add a conversion just to be sure that future updates of the TOS will be displayed correctly if they'll include formatted text.

I've already implemented a parser for the FormattedText in the [message_bubble.rs](https://github.com/melix99/telegrand/blob/a22e38928da6b7843706841bbfa778a678e4d0e2/src/session/content/message_bubble.rs#L173) so I'd day to move that functions to utils.rs and use them to convert the text to markup, this way the eventual formatted text should be displayed correctly in the dialog.